packages/agents/sequencer/src/config.ts

```diff
@@ -95,6 +96,13 @@ export const getEnvConfig = (
       : configJson.relayers
       ? configJson.relayers
       : configFile.relayers,
+    relayerFeeTolerance: process.env.RELAYER_FEE_TOLERANCE
```

Comment:
this looks good. we may wanna add an API endpoint for retrieving this number in the near future, so users can do an accurate calc on SDK-side as to whether the sequencer will reject their transfer